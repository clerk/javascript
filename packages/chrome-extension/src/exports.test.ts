import * as publicExports from './index';

describe('public exports', () => {
  it('should not include a breaking change', () => {
    expect(Object.keys(publicExports).sort()).toMatchSnapshot();
  });
});
