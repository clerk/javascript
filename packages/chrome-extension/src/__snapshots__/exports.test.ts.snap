// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`public exports should not include a breaking change 1`] = `
[
  "AuthenticateWithRedirectCallback",
  "ClerkLoaded",
  "ClerkLoading",
  "ClerkProvider",
  "CreateOrganization",
  "MagicLinkErrorCode",
  "MultisessionAppSupport",
  "OrganizationProfile",
  "OrganizationSwitcher",
  "RedirectToCreateOrganization",
  "RedirectToOrganizationProfile",
  "RedirectToSignIn",
  "RedirectToSignUp",
  "RedirectToUserProfile",
  "SignIn",
  "SignInButton",
  "SignInWithMetamaskButton",
  "SignOutButton",
  "SignUp",
  "SignUpButton",
  "SignedIn",
  "SignedOut",
  "UserButton",
  "UserProfile",
  "WithClerk",
  "WithSession",
  "WithUser",
  "__internal__setErrorThrowerOptions",
  "isClerkAPIResponseError",
  "isKnownError",
  "isMagicLinkError",
  "isMetamaskError",
  "useAuth",
  "useClerk",
  "useMagicLink",
  "useOrganization",
  "useOrganizationList",
  "useOrganizations",
  "useSession",
  "useSessionList",
  "useSignIn",
  "useSignUp",
  "useUser",
  "withClerk",
  "withSession",
  "withUser",
]
`;
